### PR TITLE
EFX-142: add Companies section on Landing page

### PR DESCRIPTION
**Pull request by ainursharaev**
## Visual references

image
image
image
image

### CODE REVIEW

File: beta/src/components/Landing/Companies.astro [hunk 1]
```diff
@@ -0,0 +1,233 @@
...
+  }
+];
+
+
+const { lang } = getPathParamsFromId(Astro.url.pathname);
+
+---
+
+<section class="companies">
+  <header>
```
Comment (sergeysova) [inline beta/src/components/Landing/Companies.astro:134]:
```suggestion
  <h3>
```

File: beta/src/components/Landing/Companies.astro [hunk 2]
```diff
@@ -0,0 +1,233 @@
...
+    <h1>
+      {getTextLocalized(translations.Landing.Companies.title, lang)}
+    </h1>
+  </header>
+  <ul class="companies-list">
+    {companies.map(({ url, alt, yOffset, scale }, i) => {
+      return (
+        <li
+          key={i}
```
Comment (sergeysova) [inline beta/src/components/Landing/Companies.astro:143]: Are you need key in Astro components?

File: beta/src/components/Landing/Companies.astro [hunk 3]
```diff
@@ -0,0 +1,233 @@
...
+  :root {
+    --logos-bg: hsla(var(--color-stone-10), 1);
+    --logos-text: hsla(var(--color-gray-10), 1);
+    --logos-text-light: hsla(var(--color-gray-30), 1);
+  }
+
+  .companies {
+    @apply px-5 pt-12 pb-20;
+    background-color: var(--logos-bg);
+    --maxWidth: 90vw;
```
Comment (sergeysova) [inline beta/src/components/Landing/Companies.astro:172]: Sometimes ‘param-case’ sometimes ‘camelCase’

**Review by sergeysova — CHANGES_REQUESTED** [review]

File: beta/src/components/Landing/Companies.astro (hunk 2 above)
Comment (ainursharaev) [reply beta/src/components/Landing/Companies.astro:143]: good catch, didn't know that 👍

File: beta/src/components/Landing/Companies.astro (hunk 1 above)
Comment (ainursharaev) [reply beta/src/components/Landing/Companies.astro]: done ✅

File: beta/src/components/Landing/Companies.astro (hunk 2 above)
Comment (ainursharaev) [reply beta/src/components/Landing/Companies.astro]: removed ✅

File: beta/src/components/Landing/Companies.astro (hunk 3 above)
Comment (ainursharaev) [reply beta/src/components/Landing/Companies.astro]: changed ✅

**Comment by sergeysova** [comment]
@ainursharaev Thank you!

**Pull request merged**